### PR TITLE
fix: invalid workflow graph with triggers

### PR DESCRIPTION
**Pull request by OronNadiv**
## Context
Nodes often disappear when showing triggers.

image

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by jithine — APPROVED** [review]

File: app/pipeline/events/controller.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { inject as service } from '@ember/service';
 import { get, computed } from '@ember/object';
 import { jwt_decode as decoder } from 'ember-cli-jwt-decode';
 import { alias } from '@ember/object/computed';
-
+import _ from 'underscore';
```
Comment (DekusDenial) [inline app/pipeline/events/controller.js:6]: meant `lodash`?

Comment (OronNadiv) [reply app/pipeline/events/controller.js:6]: Yes! Thanks!

File: package.json [hunk 2]
```diff
@@ -135,10 +135,10 @@
     "jstree": "^3.3.7",
     "lint-staged": "^8.1.7",
     "loader.js": "^4.7.0",
+    "lodash": "^4.17.20",
```
Comment (DekusDenial) [inline package.json:138]: wanna go module by requiring `lodash.uniqby` instead?

Comment (OronNadiv) [reply package.json:138]: Sure.

**Review by DekusDenial — APPROVED** [review]

File: app/event/serializer.js [hunk 3]
```diff
@@ -1,21 +1,20 @@
 import { assign } from '@ember/polyfills';
 import DS from 'ember-data';
-import _s from 'underscore.string';
 
 export const sortWorkflowGraph = workflowGraph => {
+  const compare = (s1, s2) =>
```
Comment (jithine) [inline app/event/serializer.js:5]: also move it out of this function to top level ?, Think this need not be defined for each fn call

**Review by jithine — APPROVED** [review]

**Review by DekusDenial — APPROVED** [review]

**Pull request merged**